### PR TITLE
Fix some typos and use <code> instead of <tt>

### PR DESCRIPTION
**Pull request by jaw111**

### CODE REVIEW

**Comment by namedgraph** [comment]
There's a PR converting this to Markdown: https://github.com/w3c/sparql-12/pull/170

File: SEP/SEP-0007/sep-0007.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ This SEP describes the issues and proposes solutions.
 <p>The SPARQL 1.1 algebra operation
 &ldquo;<a href="https://www.w3.org/TR/sparql11-query/#defn_substitute">substitute</a>&rdquo; evaluates
 a graph pattern where there are specific values for variables given by a solution
-mapping. The operation is used to in the evaluation of <tt>EXISTS</tt> and
```
Comment (JervenBolleman) [inline SEP/SEP-0007/sep-0007.md:28]: Nothing down this change. In case the Markdown conversion is accepted to redo it in that version.

**Comment by JervenBolleman** [comment]
Picking this change first. Then ```sed -e 's|<code>|`|g' -e 's|</code>|`|g' SEP-0007/sep-0007.md``` and then add my work after should be the least manual work.

File: SEP/SEP-0007/sep-0007.md [hunk 2]
```diff
@@ -314,16 +314,16 @@ is only applied to variables not visible outside the sub-query.</p>
...
 notes 12 and 13.</p>
 <p>This proposal adds the restriction that any variables in a current
 row, the set of variables
 <a href="https://www.w3.org/TR/sparql11-query/#variableScope">in-scope</a>
-of the expression containing EXISTS, can not be assigned with the <tt>extend</tt>
-algebra function linked to the <tt>AS</tt> syntax.</p>
-<p>In addition, any use of <tt>VALUES</tt> in the EXISTS expression must not
+of the expression containing <code>EXISTS</code>, can not be assigned with the <code>extend</code>
+algebra function linked to the <code>AS</code> syntax.</p>
+<p>In addition, any use of <code>VALUES</code> in the <code>EXISTS</code> expression must not
```
Comment (afs) [inline SEP/SEP-0007/sep-0007.md:326]: Additional.

File: SEP/SEP-0007/sep-0007.md [hunk 3]
```diff
@@ -389,7 +389,7 @@ define the <dfn>Evaluation of Exists</dfn> <tt>exists(X)</tt>
 <h2 id="addressing-issues">Addressing Issues</h2>
 <p>This section addresses each issue identified, given the proposal above.</p>
 
-<h3 id="issue-1-some-uses-of-exists-are-not-defined-during-evaluation">Issue 1: Some uses of EXISTS are not defined during evaluation</h3>
+<h3 id="issue-1-some-uses-of-exists-are-not-defined-during-evaluation">Issue 1: Some uses of <code>EXISTS</code> are not defined during evaluation</h3>
```
Comment (afs) [inline SEP/SEP-0007/sep-0007.md:392]: Additional.

File: SEP/SEP-0007/sep-0007.md [hunk 4]
```diff
@@ -259,8 +259,8 @@ that name is not used anywhere else in the query, and the same results are
 obtained in the sub-query. A sub-query always has a projection as its top-most
 algebra operator.</p>
 <p>To preserve this, any such variables are renamed so they do not coincide with
-variables from the current row being filtered by <tt>EXISTS</tt>.</p>
-<p>The SPARQL algebra &ldquo;project&rdquo; operator has two components, an algebra expression
+variables from the current row being filtered by <code>EXISTS</code>.</p>
```
Comment (afs) [inline SEP/SEP-0007/sep-0007.md:262]: Additional.

File: SEP/SEP-0007/sep-0007.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ process that addresses these problems based on the concept of
 <h2 id="summary">Summary</h2>
 <p>The fundamental problem is that a variable can not be simply replaced by a value
 (an RDF Term) in all places in a graph pattern. There are some places where
-SPARQL function forms and &ldquo;AS&rdquo; assignment require a variable. The proposal here
+SPARQL function forms and <code>AS</code> assignment require a variable. The proposal here
```
Comment (afs) [inline SEP/SEP-0007/sep-0007.md:46]: Additional.

**Review by afs — COMMENTED** [review]
Notes of where this PR makes changes other than <tt> to <code>.

**Comment by afs** [comment]
Overall, it is easier to locate the improvements in this PR and transfer them to #170 because of GH color coded diffs.

This is PR #172.

Thank you @jaw111

**Comment by JervenBolleman** [comment]
@jaw111 Thank you. I am closing this PR as I believe your helpful changes have been integrated with PR #172

**Comment by TallTed** [comment]
I'm betting there's a stylesheet reason to use `<code>` instead of `<tt>`, but so far as I know, Markdown backticks get replaced with `<code>` when it gets shipped to the browser as HTML, so I'm wondering if there is a reason for using `<code>` instead of backticks as inline code wrappers?

**Comment by jaw111** [comment]
@TallTed according to MDN the `<tt>` element is deprecated and obsolete.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt

**Comment by namedgraph** [comment]
@TallTed they were in @JervenBolleman's version (#170). Maybe it hasn't been merged yet.

**Pull request closed without merging**